tests/frontend/pages/wallets/wallets.py

```diff
@@ -28,6 +28,42 @@ def create(self, name):
         wait = WebDriverWait(self.driver, 60)
         wait.until(EC.invisibility_of_element_located((By.CLASS_NAME, "v-dialog")))
 
+    def add_funded(self, name):
```

Comment:
Why wallet module is here? It is already in another PR.

Reply:
i need to create a wallet for test